### PR TITLE
fix: move/movel throws exceptions for negative values

### PR DESCRIPTION
**Pull request by mattiabonardi**
## Description

`MOVE` `MOVEL` throws exceptions for negative values.

Related to # (issue)

## Checklist:
- [ ] There are tests regarding this feature
- [x] The code follows the Kotlin conventions (run `./gradlew ktlintCheck`)
- [x] The code passes all tests (run `./gradlew check`)
- [ ] There is a specific documentation in the `docs` directory

### CODE REVIEW

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/movel.kt [hunk 1]
```diff
@@ -159,17 +159,29 @@ private fun valueToString(value: Value, type: Type): String {
...
 
         is NumberType -> {
-            if (value is IntValue) {
-                val zeros = "0".repeat(type.numberOfDigits - s.length)
-                zeros + s
-            } else {
-                s = s.replace(".", "")
-                val zeros = "0".repeat(type.numberOfDigits - s.length)
-                zeros + s
+            when (value) {
```
Comment (lanarimarco) [inline rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/movel.kt:162]: Too much duplicated code. `IntValue` and `DecimalValue` extend `NumberValue` that provides a method `isNegative.` Follow my advice and see if you are able to make code cleaner.

**Review by lanarimarco — CHANGES_REQUESTED** [review]
Unnecessary duplicated code.

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/movel.kt (hunk 1 above)
Comment (mattiabonardi) [reply rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/movel.kt]: Fixed with isNegative() method

**Review by lanarimarco — APPROVED** [review]
Ok

**Pull request merged**